frontend/ui/src/pages/Inbox/ConversationListItem/index.tsx

```diff
@@ -38,8 +38,7 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
 
 const FormattedMessage = ({message}: FormattedMessageProps) => {
   if (message && message.content) {
-    const messageJSON = JSON.parse(message.content);
-    return <>{messageJSON.text}</>;
+    return <>{message.content.text}</>;
```

Comment:
This will break with Twilio because `message.content` is a String

Reply:
I thought this was already using the render library to be honest.